frontend/src/components/molecules/MatchWinnerDeterminationDialog/MatchWinnerDeterminationDialog.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import Button from '@material-ui/core/Button';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogContentText from '@material-ui/core/DialogContentText';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import Slide from '@material-ui/core/Slide';
+import {TransitionProps} from '@material-ui/core/transitions';
+import {Card, withStyles} from "@material-ui/core";
+
+const Transition = React.forwardRef(function Transition(
+    props: TransitionProps & { children?: React.ReactElement<any, any> },
+    ref: React.Ref<unknown>,
+) {
+    return <Slide direction="up" ref={ref} {...props} />;
+});
+
+type MatchWinnerDeterminationDialogProps = {
+    agreeCallback: (() => void),
+    isOpen: boolean
+    teamName: number | string | undefined;
+}
+
+const FullParentButton = withStyles({
+    root: {
+        position: "absolute",
+        top: 0,
+        left: 0,
+        width: "100%",
+        height: "100%",
+        opacity: 0,
+
+    }
+})(Card);
+
+export default function MatchWinnerDeterminationDialog(props: MatchWinnerDeterminationDialogProps) {
+    const [open, setOpen] = React.useState(props.isOpen);
+
+    const handleClickOpen = () => {
+        setOpen(true);
+    };
+
+    const handleClose = () => {
+        setOpen(false);
+    };
+
+    const handleAgreeAndClose = () => {
+        setOpen(false);
+        props.agreeCallback();
+    };
+
+    return (
+        <div>
+            <FullParentButton variant="outlined" color="primary" onClick={handleClickOpen}>
+            </FullParentButton>
+            <Dialog
+                open={open}
+                TransitionComponent={Transition}
+                keepMounted
+                onClose={handleClose}
+                aria-labelledby="alert-dialog-slide-title"
+                aria-describedby="alert-dialog-slide-description"
+            >
+                <DialogTitle id="alert-dialog-slide-title">{"Czy ustawić zwycięzcę meczu?"}</DialogTitle>
```

Comment:
Oooo spoko :) Moze warto gdzies obok karty z meczami dodac jakas z instrukcja jak to dziala :) ?

Reply:
Myślałem, że będzie intuicyjnie.
Ogólnie najeżdżając na team mamy hovera który pokazuje info o tym żeby awansować daną drużynę.

Chyba, że nie o to chodzi?

Reply:
Widzę, widzę, ale zanim nie najechalem przypadkiem i nie kliknalem w ogole sie nie spodziewalem, ze tak to dziala :)

Reply:
Ha, widzisz, czyli musze pomyśleć co z tym zrobić!